compiler/src/iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -1865,19 +1906,24 @@ class ExportOpConversion : public EmitCConversionPattern<IREE::VM::ExportOp> {
           emitc::PointerType::get(emitc::OpaqueType::get(ctx, argumentsType)),
           /*operand=*/argumentsData);
 
-      argumentStruct.value = arguments.getResult();
+      auto arguments_ =
+          emitc_builders::asLValue(rewriter, loc, arguments.getResult());
+
+      argumentStruct.value = arguments_;
```

Comment:
Maybe change to
```suggestion
      auto argumentsLValue_ =
          emitc_builders::asLValue(rewriter, loc, arguments.getResult());

      argumentStruct.value = argumentsLValue_;
```
for consistency with the other renamings?